docs/guides/func_yaml.md

```diff
@@ -51,17 +51,40 @@ envs:
 - value: '{{ configMap:myconfigmap2 }}'     # (4) all key-value pairs in ConfigMap as env variables
 ```
 
-### `volumes`
-Kubernetes Secrets or ConfigMaps can be mounted to the function as a Kubernetes Volume accessible under specified path. Below you can see an example how to mount the Secret `mysecret` to the path `/workspace/secret` and the ConfigMap `myconfigmap` to the path `/workspace/configmap`. This Secret/ConfigMap needs to be created before it is referenced in a function.
+### `image`
```

Comment:
> I do understand that alphabetical order may make sense in here, but shouldn't we instead sort the section of this docs as the fields are really sorted in `func.yaml`?

Personally I would prefer alphabetical, since that's more intuitive if you don't have `func.yaml` open.